features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/multi-dc/OGG/oracle/tables.sql

```diff
@@ -302,8 +303,8 @@ CREATE TABLE UM_USER_PERMISSION (
                     UM_IS_ALLOWED SMALLINT NOT NULL,
                     UM_TENANT_ID INTEGER DEFAULT 0,
                     UNIQUE (UM_PERMISSION_ID, UM_USER_NAME, UM_TENANT_ID),
-                    FOREIGN KEY (UM_PERMISSION_ID, UM_TENANT_ID) REFERENCES UM_PERMISSION(UM_ID, UM_TENANT_ID),
-                    --FOREIGN KEY (UM_USER_ID) REFERENCES UM_USER(UM_ID),
+                    FOREIGN KEY (UM_PERMISSION_ID, UM_TENANT_ID) REFERENCES UM_PERMISSION(UM_ID, UM_TENANT_ID) ON DELETE CASCADE,
+                    --FOREIGN KEY (UM_USER_ID) REFERENCES UM_USER(UM_ID) ON DELETE CASCADE,
```

Comment:
We have to remove all on delete cascades and they should be replaced with triggers

Reply:
Please check other occurences as well